### ISSUE TITLE
Log-scale in Params and Scatters Explorers

### ISSUE THREAD

**Issue by sirykd** [aimhubio/aim#2500]
## 🚀 Feature

It would be very useful to have a log-scale option in Params and Scatters Explorers.
Ideally, the scale could be switched on a per-param/per-axis basis (similarly to #2358).

### Motivation

Log scale may be more convenient for visual analysis for some commonly used hyperparameters, such as learning rate or weight decay.

### Pitch

A toggle/button/menu to switch individual parameters/axes to log-scale in Params and Scatters Explorers.

**Comment by roubkar** [comment]
Hey @sirykd , thanks for opening this issue.
Will add to our roadmap.
